builtin/mainmenu/tab_content.lua

```diff
@@ -87,9 +89,17 @@ local function get_formspec(tabview, name, tabdata)
 			desc = info.description
 		end
 
+		local title_and_name
+		if selected_pkg.type == "game" then
+			title_and_name = selected_pkg.name
+		else
+			title_and_name = (selected_pkg.title or selected_pkg.name) ..
+				core.colorize("#BFBFBF", "\n(" .. selected_pkg.name .. ")")
```

Comment:
(no visible difference, just cleaner)
```suggestion
			title_and_name = (selected_pkg.title or selected_pkg.name) .. "\n" ..
				core.colorize("#BFBFBF", "(" .. selected_pkg.name .. ")")
```